docs/modules/ROOT/pages/explanations/decisions/deployments.adoc

```diff
@@ -0,0 +1,69 @@
+= Decision How to Deploy Compositions and XRDs
+
+== Problem
+
+We need to deploy XRDs and Compositions for AppCat to Kubernetes clusters.
+Also writing the Compositions and XRDs plainly is somewhat a hassle (yaml engineering).
+Our initial Redis POC contained almost 400 lines of yaml.
+Most of those lines consist of patch operations and many of those are very similar to one another.
+This makes maintaining and understanding the compositions very hard.
+Having helpers with engineering Compositions and XRDs would be a welcomed bonus for the solution.
+
+== Requirements
+
+* Manage XRDs
+* Manage Compositions
+* Manage Providers and their config
+* Promote re-usability were possible
+* Help with developing XRDs and Compositions
+* Should support Crossplane packages in the future
+* Should support APPUiO Managed in the future
+
+== Proposals
+
+=== Proposal 1: Fine Grained Commodore Packages
+
+This proposal will enhance https://github.com/projectsyn/component-crossplane[`+component-crossplane+`] in such a way that it can also deploy XRDs and Compositions via parameters.
+The definitions for the XRDs and Compositions will live in the https://syn.tools/commodore/tutorial/package.html[Commodore Packages].
```

Comment:
I thought the definitions of the XRDs and Compositions will be saved in a new component and from there we pick a relevant XRD and Composition for specific package. Is it possible to get these objects in a package from outside a component jsonnet files?

Reply:
Having the XRDs and Compositions in a component is in proposal 2.